xml/System/Type.xml

```diff
@@ -11335,8 +11342,8 @@ GetType(Array).IsAssignableFrom(type)
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value indicating whether the current <see cref="T:System.Type" /> represents a type parameter in the definition of a generic method.</summary>
+        <value><see langword="true" /> if the <see cref="T:System.Type" /> object represents a type parameter of a generic method definition; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the <see cref="T:System.Type" /> object represents a type parameter of a generic method definition; otherwise, <see langword="false" />.</value>
```